src/js/helpers/graph/declarationHelper.mock.js

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const singleNode = {
```

Comment:
I would propose simple convention:
node ids: `A`, `B`, `C`
subtasks ids: `A-A`, `A-B`, `A-C`
subtasks of subtasks ids: `A-A-A`

Then all labels can be `${nodeId} label`.

Reply:
I settled on `A-A` for subtasks and `A A` for children to easily differentiate nested composites